build/parse.y

```diff
@@ -442,6 +457,19 @@ expr:
                         Else: $5,
                 }
 	}
+| _DEF _IDENT '(' exprs_opt ')' ':' block
```

Comment:
does it support single-line function definition?
def f(): return 1

if not, add a TODO

Reply:
Done.